graphene_federation/main.py

```diff
@@ -20,9 +20,11 @@ def build_schema(query, mutation=None, **kwargs):
     return graphene.Schema(query=_get_query(schema, query), mutation=mutation, **kwargs)
 
 
-def key(fields: str):
+def key(fields: str, *args: str):
```

Comment:
I think the more explicit and consistent way would be to use new decorator for each primary key
as in federation docs
https://www.apollographql.com/docs/apollo-server/federation/advanced-features/#multiple-primary-keys
```
type Product @key(fields: "upc") @key(fields: "sku") {
  upc: String!
  sku: String!
  price: String
}
```
otherwise, we should make breaking changes and set fields as args:
`def key(*fields: str)`
Cause having `*args` together with `fields` is a bit confusing